docs/cog_guides/filter.rst

```diff
@@ -0,0 +1,303 @@
+.. _filter:
+
+======
+Filter
+======
+
+This is the cog guide for the filter cog. You will
+find detailed docs about usage and commands.
+
+``[p]`` is considered as your prefix.
+
+.. note:: To use this cog, load it by typing this::
+
+        [p]load filter
+
+.. _filter-usage:
+
+-----
+Usage
+-----
+
+This cog is designed for "filtering" unwanted words and phrases from a guild.
```

Comment:
```suggestion
This cog is designed for "filtering" unwanted words and phrases from a server.
```
Everywhere else in this document, "server" is used, so either consistently use "guild" or change this instance.